### PR TITLE
Added docstring for is_open method in sets.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hi. 

```is_open``` property method was missing  a docstring.

This has been now added.

Thank you.

### CODE REVIEW

**Comment by asmeurer** [comment]
The tests are failing from tailing whitespace. You can check this on your own computer before committing with `./bin/test quality`.

File: sympy/sets/sets.py [hunk 1]
```diff
@@ -455,6 +455,19 @@ def boundary(self):
...
     @property
     def is_open(self):
+        """
+        Property method to check whether a set is open.
+        If Intersection is not found, return True. Otherwise,
+        return None because we can't confidently claim that
+        intersection exists.
+
+        Examples
+        ========
```
Comment (asmeurer) [inline sympy/sets/sets.py]: Empty line after the `========`

Comment (Newman101) [inline sympy/sets/sets.py]: Sure, I'll fix this tomorrow.

**Comment by Newman101** [comment]
That's sorted out now.

**Pull request merged**